eslint.config.js

```diff
@@ -0,0 +1,37 @@
+import hypothesis from 'eslint-config-hypothesis';
+import jsxA11y from 'eslint-plugin-jsx-a11y';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  {
+    ignores: [
+      '.tox/**/*',
+      '.yalc/**/*',
+      '.yarn/**/*',
+      'build/**/*',
+      '**/vendor/**/*.js',
+      '**/coverage/**/*',
+      'docs/_build/*',
```

Comment:
I wouldn't suggest this change in this PR, but I note the ESLint docs mention [a utility](https://eslint.org/docs/latest/use/configure/ignore#including-gitignore-files) from the `@eslint/compat` package as an option for respecting `.gitignore`.